util/modlog.js

```diff
@@ -58,46 +58,32 @@ module.exports = class ModLog {
             .setAuthor(this.moderator.tag, this.moderator.avatar)
             .setColor(ModLog.colour(this.type))
             .setDescription([
-                `**Type**: ${this.type}`,
+                `**Type**: ${this.type[0].toUpperCase() + this.type.slice(1)}`,
                 `**User**: ${this.user.tag} (${this.user.id})`,
-                `**Reason**: ${this.reason || `Use \`${this.guild.configs.prefix}reason ${this.case} to claim this log.\``}`
+                `**Reason**: ${this.reason || `Use \`${this.guild.configs.prefix}reason ${this.case}\` to claim this log.`}`
             ])
             .setFooter(`Case ${this.case}`)
             .setTimestamp();
         return embed;
     }
 
-    // Here we get the case number and create a modlog provider entry
-
     async getCase() {

```

Comment:
useless

Reply:
I accidentally removed that, I've put it back in.

Reply:
useles async...